compiler/tflchef/core/src/OpChef.h

```diff
@@ -45,4 +45,14 @@ struct OpChefFactory
   virtual std::unique_ptr<OpChef> create(const tflchef::Operation *operation) const = 0;
 };
 
+static void CheckCustomOp(const tflchef::Operation operation, std::string op_type)
```

Comment:
plz do not use `static` in header. there will be duplicate static functions across the callers.

Reply:
@seanshpark 
Oh... I didn't know that part, Thank you for letting me know.
I don't know if I should create an `OpChef.cpp` file as an implementation for the `OpChef.h` header and
put the function in it.
Could you advise me where it would be good...?

Reply:
> Could you advise me where it would be good...?

Please think of what the purpose of this function, and the scope of the functionality.

And looking in other global methods in `Convert.h`, please follow existing snake_case convention.

And does `CheckCustomOp` express the purpose of new method?

Naming is difficult and needs some experience, which may require try and error.